integrationtests/config/rho_loss.yaml

```diff
@@ -0,0 +1,74 @@
+pipeline:
+  name: RHO-Loss Test Pipeline
+  description: Example pipeline
+  version: 1.0.0
+model:
+  id: ResNet18
+  config:
+    num_classes: 10
+model_storage:
+  full_model_strategy:
+    name: "PyTorchFullModel"
+training:
+  gpus: 1
+  device: "cpu"
+  dataloader_workers: 2
+  use_previous_model: False
+  initial_model: random
+  batch_size: 4
+  shuffle: False
+  optimizers:
+    - name: "default"
+      algorithm: "SGD"
+      source: "PyTorch"
+      param_groups:
+        - module: "model"
+          config:
+            lr: 0.1
+            momentum: 0.001
+  optimization_criterion:
+    name: "CrossEntropyLoss"
+  checkpointing:
+    activated: False
+data:
+  dataset_id: image_test_dataset
+  transformations: ["transforms.ToTensor()",
+                    "transforms.Normalize((0.1307,), (0.3081,))"]
+  bytes_parser_function: |
+    from PIL import Image
+    import io
+    def bytes_parser_function(data: memoryview) -> Image:
+      return Image.open(io.BytesIO(data)).convert("RGB")
+trigger:
+  id: DataAmountTrigger
+  num_samples: 10
+selection_strategy:
+  name: CoresetStrategy
+  maximum_keys_in_memory: 1000
+  storage_backend: "database"
+  limit: -1
+  tail_triggers: 0
+  downsampling_config:
+    strategy: RHOLoss
+    sample_then_batch: False
+    holdout_set_ratio: 30
+    ratio: 60
+    il_training_config:
+      il_model_id: ResNet18
```

Comment:
The github runners are super duper slow and i dont want to overload them. In modyn/models there should be a dummy model with just a linear layer. Can we use this here for both the IL and regular model? That minimizes the used computation

Reply:
Make sense I will replace it with Dummy!

Reply:
You are right after I replaced ResNet18 with Dummy the test becomes snappy fast!

This comment was addressed.